src/08-tasks/tasks.apib

```diff
@@ -11,8 +11,8 @@ Get a list of tasks.
             + ids: `9c475de2-1e99-0328-bc19-271b27e921d8`,`6325b4e5-c6a6-060b-b010-b81892e82fa7` (array[string], optional)
             + user_id: `f29abf48-337d-44b4-aad4-585f5277a456` (string, optional) - Returns Tasks that are assigned to this user or to a team to which this user belongs.
             + completed: true (boolean, optional)
-            + due_after: `2019-02-04` (string, optional)
-            + due_before: `2019-10-20` (string, optional)
+            + due_after: `2019-02-04` (string, optional, nullable)
+            + due_before: `2019-10-20` (string, optional, nullable)
```

Comment:
B: I don't think these filters should be `nullable`, for me that would mean "give me tasks with no due date" but due date is a required field of a Task.